libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -110,8 +110,10 @@ fun asReceivePaymentRequest(reqData: ReadableMap): ReceivePaymentRequest? {
     if (description != null) {
         val preimage = reqData.getArray("preimage")?.let { asUByteList(it) }
         val openingFeeParams = reqData.getMap("openingFeeParams")?.let{ asOpeningFeeParams(it) }
-
-        return ReceivePaymentRequest(amountSats.toULong(), description, preimage, openingFeeParams)
+        val useDescriptionHash : Boolean? = if (hasNonNullKey(reqData, "useDescriptionHash")) reqData.getBoolean("useDescriptionHash") else null;
+        val expiry : ULong? = if (hasNonNullKey(reqData, "expiry")) reqData.getInt("expiry").toULong() else null;
```

Comment:
This `getInt` limits the value range to 32 bits, I think its better to use `getDouble` when converting to Long/ULong.

Reply:
But we are not converting to Long/ULong we are converting to UInt.

Reply:
`expiry` is u64, so needs `toULong()`

Reply:
Right, I confused that with cltv. Fixed.